modules/confirm-email/confirmation-email.service.js

```diff
@@ -0,0 +1,21 @@
+class ConfirmationEmailService {
+  async confirmEmail(
+    user,
+    confirmationMessage,
+    sendEmail,
+    firstName,
+    token,
+    language
+  ) {
+    const message = {
+      from: process.env.MAIL_USER,
+      to: user.email,
+      subject: '[HORONDI] Email confirmation',
```

Comment:
Ideally in `EmailService` only `from` should be defined here because it is going to be the same for any email sending action in our application. Other fields should be provided from the config object from the outside

Reply:
solved